build-base.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# Building an debian base image
+
+if [[ ! -f .env ]]; then
+    echo "Default build arguments don't exists. Creating one from default value."
+    cp .example.env .env
+fi
+
+docker-compose -f docker-compose.build.yml build postgis-base
+
```

Comment:
I moved this from `base_build` and adapt it to use docker-compose, multi-stage build and the `.env` file for build args